src/main/java/org/moltimate/moltimatebackend/config/SwaggerConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.moltimate.moltimatebackend.config;
+
+import com.google.common.base.Predicates;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.ResponseEntity;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.time.LocalDate;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfiguration {
+
+    @Bean
+    public Docket moltimateApi() {
```

Comment:
Did we confirm that this still works after moved?

Reply:
Yes, it still works